src/main/kotlin/org/radarbase/gateway/inject/PushIntegrationEnhancerFactory.kt

```diff
@@ -12,22 +10,14 @@ import org.radarbase.push.integration.common.inject.PushIntegrationResourceEnhan
 
 class PushIntegrationEnhancerFactory(private val config: Config) : EnhancerFactory {
 
-    val authConfig = AuthConfig(
-        managementPortal = MPConfig(
-            url = config.auth.managementPortalUrl
-        ),
-        jwtResourceName = config.auth.resourceName,
-        jwtIssuer = config.auth.issuer
-    )
-
     override fun createEnhancers(): List<JerseyResourceEnhancer> {
 
         val enhancersList = mutableListOf(
             GatewayResourceEnhancer(config),
-            ConfigLoader.Enhancers.radar(authConfig),
             ConfigLoader.Enhancers.health,
             ConfigLoader.Enhancers.httpException,
             ConfigLoader.Enhancers.generalException,
+            RadarResourceEnhancer(),
```

Comment:
Not sure about this; why not use the `RadarJerseyResourceEnhancer` add the default Auth implementation like this does: https://github.com/RADAR-base/radar-jersey/tree/master/radar-jersey/src/main/kotlin/org/radarbase/jersey/auth/disabled?

Reply:
Because the `RadarJerseyResourceEnhancer` needs AuthConfig which we do not have anymore. We do indeed return DisabledAuth in the `GarminAuthValidator.verify(...)` method.